python-sdk/examples/cb_integration_test_2.py

```diff
@@ -115,12 +117,17 @@ def execute_queries(client, file):
         # user-id,group-id
 
         response = client.send_query({"user": data[0], "groupId": data[1]})
-        #time.sleep(0.5)
+        # time.sleep(0.5)
 
         print("Result: \"\nuser\": " + data[0] + response.json_body)
         count += 1
 
+
 if __name__ == '__main__':
+
+    user_id = '8c24137d-24ed-498d-a5d6-f21051db9ce5'
+    user_secret = 'sILCF14ljZZdnqbzzg9SDJemz6wDY3YMH1rdbZ4r7RLzsA2vIh5R9H68WSMJN6ry'
```

Comment:
these are created by the server, how can you hard code them? You must be assuming some previous session and these are already in the DB? A clean example would take these are parameters on the CLI.